serveradmin/api/models.py

```diff
@@ -0,0 +1,18 @@
+import hashlib
+
+from django.db import models
+
+
+class Lock(models.Model):
+    class Meta:
+        db_table = 'api_lock'
+
+    hash_sum = models.CharField(max_length=40, null=False, unique=True)
+    until = models.DateTimeField(null=True)
+    duration = models.PositiveIntegerField(null=True)
+
+    @staticmethod
```

Comment:
I would suggest to define it as a `@classmethod` so it is overridable for a super class.